### ISSUE TITLE
Strange Error

### ISSUE THREAD

**Issue by Wulfheart** [beyondcode/invoker-community#218]
------
**OS**: linux
**Invoker Version**: 2.5.1
**Laravel Version**: 8.47.0
**Local project**: unknown
**PHP Binary**: /usr/local/bin/php
**Route**: /project/index?class=App%5CModels%5CUser&time=1624139966351&project=remote-nRETEyinCnzWyLgj1sZqYc

**Error**:
```
In MySqlDriver.php line 8:
                                                              
  Class "Doctrine\DBAL\Driver\AbstractMySQLDriver" not found
```

**Comment by devingray** [comment]
Please see here: #214 

`composer require doctrine/dbal` will help to solve in mean time

**Comment by mpociot** [comment]
This is now fixed in Invoker 2.5.2
